pipeline: False     # If the node should be run as a pipeline (just process the data)
visualize: True     # Toggle to visualise the data
debug: True         # Toggle to enable debug features
remove_non_labled: True  # Removes all poses that are not associated with a collision state


# RC controll state switch
state_topic: "/squash/logging_state"    # Toppic for the collision state of the robot, number correspond to the values of the RC channel
state_pos_label: 0
state_neg_label: 2
state_neutral_label: 1    # Neutal label is used to reset or disengage when a collision occured

# Conversion of input to output labels
out_pos_label: 1  
out_neg_label: 0
out_neutral_label: -1

# Parameters that need to be set
output_file: ""
traj_file: "/data/data_processed/2024_04_12_00_58_08Z/raw_data/odom_wildcat_traj.txt"
rc_bag_files: ["/data/data_processed/2024_04_12_00_58_08Z/raw_data/rc_states.bag"]
